EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -198,6 +220,18 @@ protected void onUpdate(AjaxRequestTarget target) {
         layoutForm.add(dropDown);
     }
 
+    private StringResourceModel getAttributeInputDescription(AttributeInput attributeInput) {
+        StringResourceModel inputDescriptionModel = new StringResourceModel("description.${type}", this,
+                new Model<TypeTreeNode>(attributeInput.getAttribute()));
+        if (attributeInput.hasDifferentMethods()) {
+            IModel<AttributeInput> attributeInputIModel = new Model<AttributeInput>(attributeInput);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2293)